frontend/src/components/EventsGrid /EventsGrid.tsx

```diff
@@ -0,0 +1,15 @@
+import EventCard from "../EventCard/EventCard";
+
+type EventsGridProps = {
+  events: any;
```

Comment:
Again, you should not use any. Instead, write a type and verify that the data that you're receiving is the expected one.